src/transformers/pipelines/object_detection.py

```diff
@@ -83,7 +83,7 @@ def __call__(self, *args, **kwargs) -> Union[Predictions, List[Prediction]]:
 
                 The pipeline accepts either a single image or a batch of images. Images in a batch must all be in the
                 same format: all as HTTP(S) links, all as local paths, or all as PIL images.
-            threshold (`float`, *optional*, defaults to 0.9):
+            threshold (`float`, *optional*, defaults to 0.5):
```

Comment:
Should we also update in postprocess?
https://github.com/huggingface/transformers/blob/1872bde7fc6a5d6796bd742bc2dc38eaf8069c5d/src/transformers/pipelines/object_detection.py#L123

Reply:
Haha sorry I updated the wrong code :D thanks for spotting